webapp/channels/src/components/announcement_bar/notification_permission_bar/notification_permission_unsupported_bar.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useState} from 'react';
```

Comment:
Those ARIA attributes aren't actually needed as long as it's using a proper button or anchor element (although we use the wrong element) and the element contains text which I think it does in this case.

As for the focus, I don't think that should be moved here automatically since I think the post textbox is still the more important thing that the user would want to be focused on, and moving the user's focus unexpectedly can cause accessibility issues. I just learned about [the status role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role) which is seems like the announcement bar should use, but that's outside the scope of this PR and I want to read up more on that to make sure we don't do it wrong